Libraries/LibGfx/ImageFormats/PNGLoader.cpp

```diff
@@ -187,12 +193,24 @@ ErrorOr<void> PNGImageDecoderPlugin::initialize()
     png_set_filler(m_context->png_ptr, 0xFF, PNG_FILLER_AFTER);
     png_set_bgr(m_context->png_ptr);
 
-    char* profile_name = nullptr;
-    int compression_type = 0;
-    u8* profile_data = nullptr;
-    u32 profile_len = 0;
-    if (png_get_iCCP(m_context->png_ptr, m_context->info_ptr, &profile_name, &compression_type, &profile_data, &profile_len))
-        m_context->icc_profile = TRY(ByteBuffer::copy(profile_data, profile_len));
+    png_byte color_primaries;
+    png_byte transfer_function;
+    png_byte matrix_coefficients;
+    png_byte video_full_range_flag;
```

Comment:
We should be in the habit of initializing plain types
```c++
png_byte color_primaries { 0 };
```

Reply:
My bad, I know that, it's just an oversight on my side.